Packs/PaloAltoNetworks_IoT3rdParty/Scripts/GeneratePANWIoTDeviceTableQueryForServiceNow/GeneratePANWIoTDeviceTableQueryForServiceNow.yml

```diff
@@ -0,0 +1,28 @@
+args:
+- description: device inventory list
+  isArray: true
+  name: devices
+  required: true
+comment: This script is to generate a single query or query list to query for ServiceNow
+commonfields:
+  id: GeneratePANWIoTDeviceTableQueryForServiceNow
+  version: -1
+dockerimage: demisto/python3:3.8.6.13358
+enabled: true
+name: GeneratePANWIoTDeviceTableQueryForServiceNow
+outputs:
+- contextPath: PanwIot3rdParty.Query
+  description: list of query
+runas: DBotWeakRole
+runonce: false
+script: ''
+scripttarget: 0
+subtype: python3
+tags:
+- PANW IoT 3rd Party Integration
+- servicenow v2
+timeout: 600ns
```

Comment:
all other have a much bigger timeout, is this intended?

Reply:
This does not need a much bigger timeout